e2e-tests/graphql-spring-examples2/src/main/java/com/foo/graphql/examples/spring/petshop/entities/Pet.java

```diff
@@ -0,0 +1,47 @@
+package com.foo.graphql.examples.spring.petshop.entities;
+
+import lombok.Data;
```

Comment:
i am very against Lomok (long story...). it should NOT be used.
notice that, for the e2e tests, you can write them in Kotlin if you want, eg see `e2e-tests/spring-rest-postgres`